lightning/src/chain/package.rs

```diff
@@ -1240,10 +1240,8 @@ impl Readable for PackageTemplate {
 /// input amounts.
 ///
 /// If the proposed fee is less than the available spent output's values, we return the proposed
-/// fee and the corresponding updated feerate. If fee is under [`FEERATE_FLOOR_SATS_PER_KW`], we
-/// return nothing.
-///
-/// [`FEERATE_FLOOR_SATS_PER_KW`]: crate::chain::chaininterface::INCREMENTAL_RELAY_FEE_SAT_PER_1000_WEIGHT
+/// fee and the corresponding updated feerate. If fee is under [`INCREMENTAL_RELAY_FEE_SAT_PER_1000_WEIGHT`],
+/// we return nothing.
```

Comment:
Except we should still use `FEERATE_FLOOR_SATS_PER_KW` here, since we're talking about the min feerate, not the incremental feerate.

```suggestion
/// fee and the corresponding updated feerate. If fee is under [`FEERATE_FLOOR_SATS_PER_KW`],
/// we return nothing.
///
/// [`FEERATE_FLOOR_SATS_PER_KW`]: crate::chain::chaininterface::FEERATE_FLOOR_SATS_PER_KW
```